app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
```

Comment:
- [ ] You should add associations correctly in the user model, it may be something like this 👇 

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```